src/Domain/Tags.hpp

```diff
@@ -46,6 +66,23 @@ struct ElementMap : db::DataBoxTag {
                                                    VolumeDim>>;
 };
 
+/// \ingroup DataBoxTagsGroup
+/// \ingroup ComputationalDomainGroup
+/// Computes the inverse Jacobian of the map held by `MapTag` at the coordinates
+/// held by `SourceCoordsTag`. The coordinates must be in the source frame of
+/// the map.
+template <typename MapTag, typename SourceCoordsTag>
+struct InverseJacobian : db::ComputeItemTag {
+  static constexpr db::DataBoxString label = "InverseJacobian";
```

Comment:
Can this be a prefix tag so we don't get a name conflict once we have multiple maps?